apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -243,6 +243,19 @@ export default connect(
     }
 
     displayOutputToConsole() {
+      const inspectorTheme = {
+        ...chromeLight,
+        ...{
+          OBJECT_VALUE_NULL_COLOR: 'rgb(88, 92, 246)',
```

Comment:
where are these colors from? if we're using them elsewhere in our codebase, we may want to move them to a shared colors/constants file

Reply:
They're part of ace editor so they aren't anywhere else in the codebase, I just stole them from inspect element.

Reply:
okay, that makes sense -- can you add a comment with that info?